Description: Avoid hardcoded timeout in tests
 Some tests fail on slow hardware.
 .
 Better to leave it to the test environment to deal with tests hanging.
Forwarded: no
Author: Jonas Smedegaard <dr@jones.dk>
Bug-Debian: https://bugs.debian.org/851318
Bug-Debian: https://bugs.debian.org/917746
Last-Update: 2018-12-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/mocha/let.js
+++ b/test/mocha/let.js
@@ -3,7 +3,6 @@
 
 describe("let", function() {
     it("Should not produce `let` as a variable name in mangle", function(done) {
-        this.timeout(10000);
 
         // Produce a lot of variables in a function and run it through mangle.
         var s = '"use strict"; function foo() {';
--- a/test/mocha/spidermonkey.js
+++ b/test/mocha/spidermonkey.js
@@ -4,7 +4,6 @@
 
 describe("spidermonkey export/import sanity test", function() {
     it("should produce a functional build when using --self with spidermonkey", function (done) {
-        this.timeout(20000);
 
         var uglifyjs = '"' + process.argv[0] + '" bin/uglifyjs';
         var command = uglifyjs + " --self -cm --wrap SpiderUglify --dump-spidermonkey-ast | " +
--- a/test/mocha.js
+++ b/test/mocha.js
@@ -3,7 +3,7 @@
     path = require('path');
 
 // Instantiate a Mocha instance.
-var mocha = new Mocha({});
+var mocha = new Mocha({ timeout: 0 });
 
 var testDir = __dirname + '/mocha/';
 
